recipes/libdeflate/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(libdeflate REQUIRED CONFIG)
+
+add_executable(test_package ../test_package/test_package.c)
+target_link_libraries(test_package PRIVATE libdeflate::libdeflate)
```

Can you re-use the CMakeLists.txt from the test package here?

```suggestion
cmake_minimum_required(VERSION 3.1)
project(test_package)

include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup(TARGETS)

add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../test_package/
                 ${CMAKE_CURRENT_BINARY_DIR}/test_package/)
```

---

ok will give it a go